pyensembl/gene.py

```diff
@@ -73,6 +74,14 @@ def __eq__(self, other):
     def __hash__(self):
         return hash(self.id)
 
+    def __getstate__(self):
+        # Must be in order of __init__ arguments
+        return [self.id, self.name, self.contig, self.start, self.end, self.strand,
```

Comment:
With a dictionary, we start getting into the mapping between `id` and `transcript_id`, which sounded confusing to me.